modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -220,6 +221,54 @@
             .nextUntil('.section-field').hide()
     }
 
+    /*
+     * Bind validation checks on available fields, triggered when
+     * the fields is blurred or changed
+     */
+    FormWidget.prototype.bindValidation = function () {
+        let fields = this.getFieldElements(),
+            form = this.$form;
+
+        for (let field of fields) {
+            let fieldClasses = field.className.split(/\s+/).map(function (className) {
+                var fieldClass = className.match(/^([a-z\-]+)-field$/);
```

Comment:
I know it's annoying, but current convention in October JS files is no semicolons at the end of lines. Feel free to provide input on this though

Reply:
I had seen semicolons on some lines, ie: https://github.com/octobercms/october/blob/1e3e49ce08405b2b9cf1e60c9d399a1c04007ec1/modules/backend/widgets/form/assets/js/october.form.js#L324 so I figured I could get away with using them, but I will take them out to maintain consistency.